src/library_nodefs.js

```diff
@@ -19,11 +19,7 @@ addToLibrary({
     isWindows: false,
     staticInit() {
       NODEFS.isWindows = !!process.platform.match(/^win/);
-      var flags = process.binding("constants");
-      // Node.js 4 compatibility: it has no namespaces for constants
-      if (flags["fs"]) {
-        flags = flags["fs"];
-      }
+      var flags = process.binding("constants")["fs"];
```

Comment:
I was going to suggest single quotes here but it looks like this file users double-quote a lot, so maybe we leave as is.